### PR TITLE
Add a caveat about month-based date arithmetic

### PR DESCRIPTION
**Pull request by danielpunkass**
Credit to @randallmeadows on Twitter for identifying the potential for flawed date math: https://twitter.com/randallmeadows/status/629360023961997313

### CODE REVIEW

**Comment by Gryzor** [comment]
MSBOT is the new clippy

**Comment by AngeloG** [comment]
It looks like you are trying to merge some comments.

File: Frameworks/Foundation/NSCalendar.mm [hunk 1]
```diff
@@ -234,7 +234,12 @@ -(NSDateComponents*) components:(NSUInteger)unitFlags fromDate:(NSDate*)fromDate
 #define SECONDS_PER_HOUR    (SECONDS_PER_MIN * 60.0)
```
Comment (dainawilson) [inline Frameworks/Foundation/NSCalendar.mm]: Dumb

**Comment by keluo-ms** [comment]
This was fixed with: https://github.com/Microsoft/WinObjC/commit/1cc7c83211c0b3ffa22df068aaeab93808fdd3c0#diff-f87d382f3259d317dec307ccc77f41a4

**Pull request closed without merging**